scripts/common.py

```diff
@@ -297,18 +297,22 @@ def open_new_tab_in_browser_if_possible(url):
     input()
 
 
-def get_remote_alias(remote_url):
-    """Finds the correct alias for the given remote repository URL."""
+def get_remote_alias(remote_urls):
+    """Finds the correct alias for the given remote repository URLs."""
     git_remote_output = subprocess.check_output(
         ['git', 'remote', '-v']).decode('utf-8').split('\n')
     remote_alias = None
-    for line in git_remote_output:
-        if remote_url in line:
-            remote_alias = line.split()[0]
+    remote_url = None
```

Comment:
Do we need this line?

Reply:
Yes, this is required since pylint gives an error that loop variable might be undefined given the input might not be a list. We can fix this if we had typings in place but initialising the variable with None is the other workaround: (https://stackoverflow.com/a/2344331.